test/nodetool/conftest.py

```diff
@@ -191,7 +191,7 @@ def cassandra_only(request):
 
 @pytest.fixture(scope="module")
 def nodetool(request, jmx, nodetool_path, rest_api_mock_server):
-    def invoker(method, *args, expected_requests=None):
+    def invoker(method, *args, expected_requests=None, expect_fail=False):
```

Comment:
I'd suggest s/expect_fail/check_returncode/ . as negative condition is always confusing. just because it takes more cycles to parse and understand it.

Reply:
makes sense , it is also  a better description of what this flag actually does.

Reply:
Done in v2.